src/auth/auth-get-domain.ts

```diff
@@ -0,0 +1,33 @@
+const thirdPartyDomainSTSUrlMappings: { [domain: string]: string; } = {
+  'bbk12.com': 'https://sts-sso.bbk12.com'
+};
+const defaultSTSUrl = 'https://s21aidntoken00blkbapp01.nxt.blackbaud.com';
+
+export class BBAuthGetDomain {
+
+  private static cacheDictionary: any = {};
+
+  public static isRegisteredDomain(domain: string = window.location.hostname): boolean {
```

Comment:
Eliminate the `domain` parameter and move it to a `public static CURRENT_DOMAIN = window.location.hostname;` property that can then be overridden by unit tests.